### PR TITLE
Add healthchecks to OpenAPI spec

### PR DESCRIPTION
**Pull request by chendrix**

### CODE REVIEW

File: openapi/openapi.yaml [hunk 1]
```diff
@@ -67,6 +67,20 @@ paths:
           $ref: "#/components/responses/BadRequest"
         "500":
           $ref: "#/components/responses/ServerError"
+  /health:
+    get:
+      parameters:
+      - $ref: "#/components/parameters/bundles"
+      - $ref: "#/components/parameters/plugins"
+      - $ref: "#/components/parameters/exclude-plugin"
+      summary: Very the server is operational
```
Comment (anderseknert) [inline openapi/openapi.yaml:76]: Very -> Verify

**Review by anderseknert — APPROVED** [review]
A typo, but LGTM. Should this be "upstreamed" to the contrib equivalent too?

**Comment by chendrix** [comment]
> A typo, but LGTM. Should this be "upstreamed" to the contrib equivalent too?

@anderseknert Yeah this should probably end up upstreamed. TBH that's a secondary concern of mine.

**Comment by anderseknert** [comment]
OK, I'll do it then.

File: .gitignore
Comment (srenatus) [inline .gitignore:1]: I'd remove this, but I don't have a strong opinion.

**Review by srenatus — APPROVED** [review]

File: .gitignore
Comment (chendrix) [reply .gitignore:1]: but you're supposed to commit .gitignores :(

**Pull request merged**